http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -1056,19 +1049,29 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
         googleServiceAccount
       ).toMap ++ req.labels
     for {
+      // Validate app type
```

Comment:
Added some extra validation: GCP supports all app types, but only Cromwell is supported on Azure.